front/src/modules/pathfinding/components/Itinerary/ItineraryV2.tsx

```diff
@@ -57,13 +61,26 @@ const ItineraryV2 = ({
     }
   };
 
+  const toastRestrictionResetWarning = () => {
```

Comment:
I think it's better if a function start with a verb, is that ok for you ?

```suggestion
  const notifyPowerRestrictionsReset = () => {
```

Reply:
Done